history/characters/42000_high_elf.txt

```diff
@@ -1254,8 +1254,15 @@
 		}
 	}
 	620.4.5={ death=yes }
+
+	603.1.1 = {
+		effect = {
+            give_job_title = job_marshal    
+        	}
+	}
```

Comment:
It's not a syntax error, but better keep dates in chronological order. Thus the 620 year goes after 603, not vice versa.